setup.cfg

```diff
@@ -1,5 +1,5 @@
 [pep8]
-exclude = ./migrations,./venv,./venv3
+exclude = ./migrations,./venv,./venv3,./venv3.6
```

Comment:
Just noticed this, it depends on what venvs are going to be named? If it's just venv we can stick with `/venv,` here and also in the `.gitignore`